docs/develop_tools/testing.md

```diff
@@ -37,7 +37,7 @@ The following is an example of a command to run all the tests:
 ## Running Tests for a Module
 
 ```bash
-./build/mvn clean test -pl :kyuubi-common
+./build/mvn clean test -pl kyuubi-common
```

Comment:
`./build/mvn clean install -pl kyuubi-common`

Reply:
And the previous Running Tests Fully should be `./build/mvn clean install`

Reply:
`./build/mvn clean test -pl kyuubi-common` works well

Reply:
yea, `kyuubi-common` works because it does not depends on `hive-jdbc-shaded`, if you try to test `kyuubi-spark-sql-engine`,  the `mvn clean test` would not not work.

Reply:
fix it